tests/nested_pandas/series/test_accessor.py

```diff
@@ -1073,3 +1073,37 @@ def test_get_list_index():
     list_index = series.array.get_list_index()
     assert len(list_index) == series.nest.flat_length
     assert np.equal(list_index, [0, 0, 0, 0, 1, 1, 1, 1]).all()
+
+
+def test_to_flatten_inner():
+    """Test .nest.to_flatten_inner()"""
+    nf = generate_data(10, 2)
+    nf["a"] = nf["a"].astype(pd.ArrowDtype(pa.float64()))
+    nf["b"] = nf["b"].astype(pd.ArrowDtype(pa.float64()))
+    nf = nf.assign(id=np.repeat(np.r_[0:5], 2))
+    nf = nf.rename(columns={"nested": "inner"})
+    nnf = NestedFrame.from_flat(nf, base_columns=[], on="id", name="outer")
+
+    # outer = nnf["outer"]
```

Comment:
remove commented out code? Or is it being kept for later?

Reply:
Thanks!